README.md

```diff
@@ -1 +1,33 @@
-# testsuite-pipelines
\ No newline at end of file
+testsuite-pipelines
+===
+This repository contains Kuadrant testsuite pipeline objects
+
+Deployment
+---
+* Install the `openshift-pipelines` Openshift operator on the cluster
+* Apply required pipeline and task resources `oc apply -k src/ -n ${PIPELINE_NAMESPACE}`
+
+Secrets
+---
+Prior to the running of the pipeline, the following resources must be created in the pipeline namespace:
+- Opaque Secret named `openshift-pipelines-credentials` containing `KUBE_PASSWORD` and `KUBE_USER` keys 
```

Comment:
Does it need to be named exactly that or can I change it for each individual test run?